File: 8.primenumbers.c
#include <stdio.h>

int main()
{
    int start = 2;
    int end = 100;
    int i, j;
    int isPrime;

    printf("All prime numbers between 1 to 100 are:\n");

    for(i=start; i<=end; i++){
        isPrime = 1; 
        for(j=2; j<=i/2; j++)
        {
            if(i%j==0)
            {
                isPrime = 0;
                break;
            }
        }
        if(isPrime==1)
        {
            printf("%d, ", i);
        }
    }
    return 0;
}

File: 8.primenumbers.cs
using System;

class Program{
    static void Main(){
      Console.WriteLine("All prime numbers between 1 to 100 are:");
      int start = 2;
      int end = 100;
      int a = 0;

      for(int i = start; i<=end; i++){
        for (int index = 1; index <= i; index++) {
            if (i % index == 0) {
               a++;
            }
         }
         if (a == 2) {
            Console.WriteLine(i);
         }
         a = 0;
      }
    }
}